principles/enterprise-architecture-principles.md

```diff
@@ -5,40 +5,23 @@ expires: 2020-01-01
-_Statement_ - The direction set out by Digital, Data and Technology (DDaT) directorates, and the path which it pursues, must always seek to solve business problems for DfE’s users – i.e. schools, teachers, providers, students, apprentices, industry, communities, partners and staff. The needs of these groups, and the DfE business areas that serve them, take primacy over other actors and constraints.
-
-_Rationale_ - Identifying, defining and subsequently solving the right user problem is key. Using architecture techniques and tools, including frameworks and roadmaps, will demonstrate clear linkage to business priorities, drivers and associated problems. DDaT will deliver the outcomes that represent user needs as capabilities for DfE business areas.
-
-_Implications_ - DDaT will pursue solutions in accordance with real user needs assuring that the path taken produces the right outcome. As a result of this principle:
-*	DDaT must cultivate a deep understanding of business area strategy, direction, aspirations, challenges and needs using a portfolio, rather than a project, view.
-*	DDaT must demonstrate understanding of user needs by using artefacts such as user stories and user journeys, to identify what is required of both DfE business and DDaT teams.
-*	DDaT must ensure that all layers of data, application and technology services and solutions are focused on delivering user needs.
-*	Solutions delivered by DDaT projects resulting in new or amended services will demonstrate user-centred outcomes.
-
-### 2. Heading
-_Statement_ - 
-
-_Rationale_ -  
-
-_Implications_ -
+Principles are general rules and guidelines, intended to be enduring and seldom amended, that inform and support the way in which the DfE fulfils its vision and principles. They reflect a level of consensus across the enterprise and embody the spirit and thinking of the enterprise architecture.
+
+If a decision is made that does not adhere to the principles an exemption needs to be sought during Logical design review either at the Architecture Review Board or Peer review.
+
+Number | Principle | Rationale (reason for adoption) | Implication (what are the consequences of conforming to this principle) | GDS service standard (SS) / Technology Code of Practice (TCoP) alignment
+- | - | - | - | -
+1 | Re-use standardised approved services/ components | Duplication of services or components adds cost and complexity. | •	 When re-using components and services, projects are likely to need to assemble several components / services to meet end-to-end requirements and not just look for single solutions.  •	 Where no approved service or component exists , any new service or component should be designed to maximise reusability. •	 Services or components should be delivered at the smallest possible granularity to facilitate re-use | TCoP8: Share, reuse and collaborate. SS2: Solve a whole problem for users. SS13: Use and contribute to open standards, common components and patterns
```

Comment:
I don't think the bullets will render as expected, may need to replace with dashes?

Reply:
Worth pasting in here to preview how it will look: https://stackedit.io/app#

Also [Markdown cheat sheet](https://www.markdownguide.org/cheat-sheet/).